### ISSUE TITLE
Move 'mark as unread' to overflow menu

### ISSUE THREAD

**Issue by cketti** [k9mail/k-9#5480]
The envelope icon seems to be confusing to many users. I suggest when showing the message view we put the "mark as unread" action into the overflow menu (by default; may be changed once #5300 is implemented).

**Comment by tobiasBora** [comment]
I agree that the picture is not really easy to understand (an arrow trying to open/close the envelope may be indeed more clear), but moving it in the overflow menu is to me a bad idea. Overflow menu is indeed hard to reach, and it would lose a lot of time going into this menu everytime I want to mark an email as unread. Another option may be to keep a better picture AND also put the item in the overflow menu.

**Comment by DasWookie** [comment]
No way! I use "unread" as a means of tracking messages I still need to do something with. As a result I regularly read a new message, decide I can't do anything about it right then and need to follow up on it later, and mark it as unread. Either leave it alone, or make it's move to overflow an option.
